README.md

```diff
@@ -424,6 +424,7 @@ Only main chapters:
 
 <p>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://keepassxc.org/"><b>KeePassXC</b></a> - store your passwords safely and auto-type them into your everyday websites and apps.<br>
+&nbsp;&nbsp;:small_orange_diamond: <a href="https://1password.com/"><b>1Password</b></a> - a secure and account syncing password manager with auto-type, 2FA and more features. <br>
```

Comment:
![README-cloudshell.txt](https://github.com/trimstray/the-book-of-secret-knowledge/files/11667891/README-cloudshell.txt)